server/lib/scraping_job_schema.js

```diff
@@ -1,23 +1,24 @@
 /**
-* Exports a mongoose schema representing a facebook job
+* Exports a mongoose schema representing a scraping job
 **/
 
 import mongoose from 'mongoose';
 import Q from 'q';
 import moment from 'moment';
 
-const facebookJobSchema = new mongoose.Schema({
-  facebook_account: String,
-  facebook_token: String,
+const scrapingJobSchema = new mongoose.Schema({
+  type: String,
+  account: String,
+  token: String,
   scraping_interval: Number, // value in minutes
   next_run: Date,
 });
 
 /**
-* Finds all the facebook jobs that should be performed
+* Finds all the scraping jobs that should be performed according to current time.
 * @returns {Promise} resolving to the list of jobs
 **/
-facebookJobSchema.statics.findJobsToRun = function findJobsToRun() {
+scrapingJobSchema.statics.findJobsToRun = function findJobsToRun() {
```

Comment:
That's a valid point @asiakaczmar :zap: I prefer the first solution. In the future I might think about implementing some kind of a timeout mechanism for forked processes, but the approach proposed in 6d31a88f6c should work fine.